apps/ios/GuideDogs/Code/App/AppContext.swift

```diff
@@ -378,7 +378,7 @@ extension AppContext {
         }
         
         static func servicesAgreementURL(for locale: Locale) -> URL {
-            return URL(string: "https://tfga.ngo/SSServices")!
+            return URL(string: "https://google.com")!
```

Comment:
This needs to remain as `https://tfga.ngo/SSServices` as it's the link to the services agreement

Reply:
Ok, will change it back right away.